product-service-ts/handlers/getProductsById.ts

```diff
@@ -0,0 +1,37 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import products from './products-mock.json';
+
+export const getProductsById: APIGatewayProxyHandler = async (event) => {
+  console.log("getProductsById FN. Event: ", event);
+  console.log("getProductsById FN. Products: ", products);
+
+  try {
+    const { id } = event?.pathParameters || {};
+    const product = products.find(product => {
+      return product.title === id;
+    });
+
+    if (!product) {
+      const error = JSON.stringify(
+        {
+          statusCode: 404,
+          body: JSON.stringify({message: `No product with such id ${id}`}),
+        }
+      );
+
+      throw new Error(error);
+    }
+
+    return {
```

Comment:
create util generateApiReponse for both handlers with the same headers,etc.

Reply:
Thanks, done.

P.S.
Created api.utils.ts.
![image](https://user-images.githubusercontent.com/8244819/97930466-14f5d880-1d74-11eb-91e4-6f3bf28828cb.png)